tests/out/collatz.spvasm

```diff
@@ -1,7 +1,7 @@
 ; SPIR-V
 ; Version: 1.0
 ; Generator: rspirv
-; Bound: 59
+; Bound: 65
```

Comment:
We don't want all outputs to show the bounds checks. let's enable it selectively for just a few, if anything, on top of the `access.wgsl`